modules/cluster-logging-json-log-forwarding.adoc

```diff
@@ -0,0 +1,48 @@
+[id="cluster-logging-json-log-forwarding_{context}"]
+= JSON log forwarding
+
+Forwarding JSON logs gives fluentd, Elasticsearch, and other JSON-aware log collectors direct access to JSON log entries as JSON objects.
```

Comment:
Should fluentd be capitalized (or backticked)? It's capitalized later (Fluentd pods) and on its docs page: https://docs.fluentd.org/

Reply:
Updated. Thanks.